projects/concurrency/README.md

```diff
@@ -0,0 +1,308 @@
+<!--forhugo
++++
+title="Concurrency"
++++
+forhugo-->
+
+In this project you're going to learn about how to safely use multiple threads. You will learn about concurrency, thread safety, and synchronisation.
```

Comment:
I know that some of this is handled in the golangbot resources you link to, but I'd probably add a very brief explanation of threads and concurrency and motivate it all by saying that most practical projects use concurrency and incorrect handling of it can result in nasty bugs or a failure to scale.